### PR TITLE
dev-tex/rubber: bump to 1.5.1

### PR DESCRIPTION
**Pull request by Flowdalic**
Signed-off-by: Florian Schmaus <flo@geekplace.eu>
Closes: https://bugs.gentoo.org/670936
Package-Manager: Portage-2.3.76, Repoman-2.3.16

### CODE REVIEW

File: dev-tex/rubber/rubber-1.5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_6 )
```
Comment (ConiKost) [inline dev-tex/rubber/rubber-1.5.1.ebuild]: What about ```python3_5``` and ```python3_7```?

File: dev-tex/rubber/rubber-1.5.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+inherit distutils-r1
+
+DESCRIPTION="A LaTeX wrapper for automatically building documents"
+HOMEPAGE="https://launchpad.net/rubber/"
+SRC_URI="https://launchpad.net/rubber/trunk/${PV}/+download/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
```
Comment (ConiKost) [inline dev-tex/rubber/rubber-1.5.1.ebuild]: If empty, drop it.

File: dev-tex/rubber/rubber-1.5.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+RDEPEND="virtual/latex-base"
+DEPEND="${RDEPEND}
+	virtual/texi2dvi"
+
+python_install() {
+	distutils-r1_python_install \
```
Comment (ConiKost) [inline dev-tex/rubber/rubber-1.5.1.ebuild]: To make it more elegant without breaking lines by ```\```, you could try to add the into an array.

Comment (Flowdalic) [reply dev-tex/rubber/rubber-1.5.1.ebuild]: If what I assume you to suggest is right, then I would prefer the current variant.

Comment (ConiKost) [reply dev-tex/rubber/rubber-1.5.1.ebuild]: That's fine, since it's only a suggestion. I you like it, keep it.

File: dev-tex/rubber/rubber-1.5.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_5 python3_6 python3_7 )
```
Comment (ConiKost) [inline dev-tex/rubber/rubber-1.5.1.ebuild]: After EAPI, there should be an empty line.

Comment (Flowdalic) [reply dev-tex/rubber/rubber-1.5.1.ebuild]: Done, also had another look at array variant and decided for it.

Comment (ConiKost) [reply dev-tex/rubber/rubber-1.5.1.ebuild]: You could shorten this to `python3_{5,6,7}`

File: dev-tex/rubber/rubber-1.5.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+PYTHON_COMPAT=( python3_5 python3_6 python3_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="A LaTeX wrapper for automatically building documents"
+HOMEPAGE="https://launchpad.net/rubber/"
+SRC_URI="https://launchpad.net/rubber/trunk/${PV}/+download/${P}.tar.gz"
+
+LICENSE="GPL-2"
```
Comment (ConiKost) [inline dev-tex/rubber/rubber-1.5.1.ebuild]: Please check the license. I see also `GPL-2+` when looking into latex modules, which are shipped insice the archive,

Comment (Flowdalic) [reply dev-tex/rubber/rubber-1.5.1.ebuild]: Indeed, changed to
```
LICENSE="GPL-2 GPL-2+"

**Comment by a17r** [comment]
lgtm, what do you think @ConiKost?

File: dev-tex/rubber/rubber-1.5.1.ebuild [hunk 6]
```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
```
Comment (ConiKost) [inline dev-tex/rubber/rubber-1.5.1.ebuild]: Maybe you could test, if python3_8 is also supported?

**Comment by ConiKost** [comment]
@a17r lgtm too.

File: dev-tex/rubber/rubber-1.5.1.ebuild (hunk 6 above)
Comment (ConiKost) [reply dev-tex/rubber/rubber-1.5.1.ebuild]: Did you test, since you approved yourself? ;-)

**Comment by a17r** [comment]
Sorry for basically forgetting about this PR :( don't hesitate to ping us next time in case of timeout
